media-gfx/blender/files/blender-4.0.1-openvdb-11.patch

```diff
@@ -0,0 +1,73 @@
+--- a/intern/cycles/scene/image_vdb.cpp	2023-11-23 14:42:38.772685628 +0100
```

Comment:
As with the other patches we have for openvdb and blender, I think it is good to include a comment explaining what the patches fixes.

I can't really tell why the three patches add in the PR are needed by just looking at the diffs.

Reply:
With openvdb-11 the API changes, and blender has to include the correct file.
First hunk does this via a macro.
Second hunk makes use of a constexpr from the openvdb headers.
Third hunk forcefully propagates a mandatory setting from bulding nanovdb with openvdb.